examples/pytorch/token-classification/sparseml_utils.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any
+
+import numpy
+
+from sparseml.pytorch.utils import ModuleExporter
+from transformers.modeling_outputs import TokenClassifierOutput
+from transformers.sparse import SparseMLTrainer
+
+
+class SparseMLTokenClassificationTrainer(SparseMLTrainer):
```

Comment:
Isn't this the same behavior as present in sparse.py? Is this needed?

Reply:
Fixed (I planned to refactor these specific trainers across tasks later, but for now just moved the compute_loss here up to SparseMLTrainer, then reused this class)